FfiSpike/FfiSpike.cpp

```diff
@@ -548,21 +550,52 @@ HANDLE start_debug_server(std::string device_identifier, std::string ddi, std::s
 	return gdb;
 }
 
-afc_connection *get_afc_connection(std::string device_identifier, const char* application_identifier, std::string method_id)
+bool start_afc_client(std::string& device_identifier, std::string& root_path, const char* application_identifier, HANDLE house_arrest_fd, std::string& method_id)
+{
+	std::string command_name = kVendDocumentsCommandName;
+	std::string::size_type index_of_documents = root_path.find(kDocumentsFolder, 0);
+	if (index_of_documents == std::string::npos)
```

Comment:
definitely not a merge-stopper, but you could extract this to a separate method called `contains` in `StringHelper.cpp`